modules/on_gke/README.md

```diff
@@ -0,0 +1,62 @@
+| forseti\_client\_vm\_ip | Forseti Client VM private IP address | string | `""` | no |
+| forseti\_cloudsql\_connection\_name | Forseti CloudSQL Connection String | string | `""` | no |
+| forseti\_server\_bucket | Forseti Server storage bucket | string | `""` | no |
+| forseti\_server\_service\_account | Forseti Server service account | string | `""` | no |
+| gke\_service\_account | The name of the IAM service account attached to the GKE cluster node-pool | string | `""` | no |
+| helm\_repository\_url | The Helm repository containing the 'forseti-security' Helm charts | string | `"https://forseti-security-charts.storage.googleapis.com/release/"` | no |
+| k8s\_forseti\_namespace | The Kubernetes namespace in which to deploy Forseti. | string | `"default"` | no |
+| k8s\_forseti\_orchestrator\_image | The container image for the Forseti orchestrator | string | `"gcr.io/forseti-containers/forseti"` | no |
+| k8s\_forseti\_orchestrator\_image\_tag | The tag for the container image for the Forseti orchestrator | string | `"latest"` | no |
+| k8s\_forseti\_server\_image | The container image for the Forseti server | string | `"gcr.io/forseti-containers/forseti"` | no |
+| k8s\_forseti\_server\_image\_tag | The tag for the container image for the Forseti server | string | `"latest"` | no |
+| k8s\_tiller\_sa\_name | The Kubernetes Service Account used by Tiller | string | `"tiller"` | no |
+| network\_policy | Apply pod network policies | string | `"false"` | no |
+| production | Whether or not to deploy Forseti on GKE in a production configuration | string | `"true"` | no |
+| project\_id | The ID of the GCP project where Forseti is currently deployed. | string | n/a | yes |
+| server\_log\_level | The log level of the Forseti server container. | string | `"info"` | no |
+
+[^]: (autogen_docs_end)
+
+## Requirements
+
+Before this module can be used on a project, you must ensure that the following pre-requisites are fulfilled:
+
+1. Terraform and kubectl are [installed](#software-dependencies) on the machine where Terraform is executed.
+2. The Service Account you execute the module with has the right [permissions](#configure-a-service-account).
+3. The Compute Engine, Kubernetes Engine, and Container Registry APIs are [active](#enable-apis) on the project you will launch the cluster in.
+4. If you are using a Shared VPC, the APIs must also be activated on the Shared VPC host project and your service account needs the proper permissions there.
+
+The [project factory](https://github.com/terraform-google-modules/terraform-google-project-factory) can be used to provision projects with the correct APIs active and the necessary Shared VPC connections.
+
+### Software Dependencies
+#### Kubectl
+- [kubectl](https://github.com/kubernetes/kubernetes/releases) 1.9.x
+#### Terraform and Plugins
+- [Terraform](https://www.terraform.io/downloads.html) 0.12
+- [Terraform Provider for GCP][terraform-provider-google] v2.9
```

Comment:
This is currently two brackets, change to make hyperlink: [Terraform Provider for GCP](terraform-provider-google)